File: DataStructures/python/queue.py
# USING LIST
q = []

# Adding elements to the queue
q.append('a')
q.append('b')
q.append('c')
q.append((1, 2))
print("Initial q")
print(q)

# Removing elements from the queue
print("\nElements deqd from queue")
print(q.pop(0))
print(q.pop(0))
print(q.pop(0))

print("\queue after removing elements")
print(q)

File: DataStructures/cpp/queue.cpp
#include <iostream>
#include <queue>

using namespace std;

int main()
{
    queue<pair<int, int>> q;
    queue<int> Q;
    q.push({5, 3});
    cout << q.size();
    q.push({53, 32});
    Q.push(3);
    Q.push(5);
    cout << "Queue size before printing the elements: " << Q.size() << endl;
    cout << "Queue element are..." << endl;
    while (!Q.empty())
    {
        cout << " " << Q.front();
        Q.pop();
    }
}